### PR TITLE
Warming stripes resolve #35

### LINKED ISSUE #35: Add tool to create stripes
Package  our small python script that creates stripes (such as https://showyourstripes.info/).

This tool will be used for our Galaxy Climate tutorial.

### PR DESCRIPTION
**Pull request by annefou**
New tool to create stripes (such as warming stripes); 
see https://www.climate-lab-book.ac.uk/2018/warming-stripes/

### CODE REVIEW

File: tools/climate-stripes/climate-stripes.xml [hunk 1]
```diff
@@ -0,0 +1,189 @@
...
+            #if str($title).strip() != ''
+                --title '$title'
+            #end if
+            #if str($adv.xname).strip() != ''
+                --xname '$adv.xname'
+            #end if
+            #if str($adv.format_date).strip() != ''
+                --xname '$adv.format_date'
+            #end if
+            #if str($adv.format_plot).strip() != ''
```
Comment (bgruening) [inline tools/climate-stripes/climate-stripes.xml]: the `!= ''` should not be needed

Comment (annefou) [reply tools/climate-stripes/climate-stripes.xml]: True! I removed it and also fix the tests! 
Thank you so much for your review.

**Comment by annefou** [comment]
> * .shed.yml file is missing
> 
> Thanks!

Ah yes! I forgot to add/commit it!!! Thanks.

File: tools/climate-stripes/climate-stripes.xml [hunk 2]
```diff
@@ -0,0 +1,190 @@
...
+**What it does**
+----------------
+
+This tools creates an image (png format) corresponding to the visualization of a timeseries as
+stripes (see https://www.climate-lab-book.ac.uk/2018/warming-stripes/).
+By default, the colormap is ``RdBu_r`` and no axis nor title are plotted. These settings can be
+changed in *Advanced settings*.
+
+
+**Usage**
```
Comment (bgruening) [inline tools/climate-stripes/climate-stripes.xml]: @annefou I'm not sure how useful this kind of help is. It looks very nerdy and in theory, all the information is attached at the parameter in the UI. It's up to you and the guidelines of the climate community :)

File: tools/climate-stripes/climate_stripes.py [hunk 3]
```diff
@@ -0,0 +1,153 @@
...
+        if self.title:
+            plt.title(self.title)
+        if self.xname:
+            nrange = self.data.index.values
+            n = int(np.floor((nrange.max() - nrange.min())/int(self.nxsplit)))
+            date_list = self.data[self.xname].loc[::n].apply(
+                             lambda x: pd.to_datetime(str(x),
+                                                      format=self.format))
+            date_list = [i.strftime(self.plot_format) for i in date_list]
+            print(date_list)
```
Comment (bgruening) [inline tools/climate-stripes/climate_stripes.py]: pint ...

File: tools/climate-stripes/climate-stripes.xml (hunk 2 above)
Comment (annefou) [reply tools/climate-stripes/climate-stripes.xml]: You are right. It does not seem necessary for Galaxy!

File: tools/climate-stripes/climate_stripes.py (hunk 3 above)
Comment (annefou) [reply tools/climate-stripes/climate_stripes.py]: Removed it! Thanks for your careful review!

**Pull request merged**